src/components/humanVsHumanGameComponent/humanVsHumanGameComponent.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import HumanComponent from './humanVsHumanGameComponent';
+import Square from '../squareComponent/squareComponent';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Human Component', () => {
+  it('renders the Human component without crashing', () => {
```

Comment:
Now that you have other working tests for the component, you can remove the "it won't crash" test